### DISCUSSION TITLE
Still impossible to use adguard quic in dnscrypt2?

### DISCUSSION THREAD

**Discussion by PerkelSimon** [DNSCrypt/dnscrypt-proxy#2314] [Q&A]
Hi, 

I try again to use adguard as quic and I see nothing in the log about that one, someone have been success with that?
thanks

**Comment by lifenjoiner** [comment]
> Supports DNS-over-HTTPS (DoH) using TLS 1.3 and QUIC

This is not saying it supports DNS over QUIC.
It means DNS over http3, which requires upgrading from http2, but not the same thing.

**Reply by PerkelSimon** [reply]
> > Supports DNS-over-HTTPS (DoH) using TLS 1.3 and QUIC
> 
> This is not saying it supports DNS over QUIC. It means DNS over http3, which requires upgrading from http2, but not the same thing.

I thought cause we can see that in toml : ` Enable *experimental* support for HTTP/3 (DoH3, HTTP over QUIC)`  we see QUIC

**Comment by jedisct1** [comment]
HTTP/3 is HTTP over QUIC.

**Reply by PerkelSimon** [reply]
> HTTP/3 is HTTP over QUIC.

OK.
But we can't use quic right?
Impossible to setup arguard
